### PR TITLE
Adds new struct bits for RegexFlag, BrandReplacement and ModelReplace…

### PR DESCRIPTION
**Pull request by elsigh**
…ment so that the new regexes.yaml at least works with the go parser

Fixes Issue #6 

But note - does not yet implement any usage of these bits in the parser.

### CODE REVIEW

**Comment by elsigh** [comment]
Refs Issue #13

File: uaparser/user_agent.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ type UserAgent struct {
 type UserAgentPattern struct {
 	Regexp            *regexp.Regexp
 	Regex             string
+	RegexFlag		  string
```
Comment (lamielle) [inline uaparser/user_agent.go]: If you care, @elsigh, it looks like you didn't run go fmt on this code, the indentation appears to be slightly off.

Not sure what your code quality standards are for this project but just thought I'd let you know.

**Comment by lamielle** [comment]
Thanks for this fix!  I'll update my code to use the new version!

File: uaparser/user_agent.go (hunk 1 above)
Comment (elsigh) [inline uaparser/user_agent.go]: Yeah, was just lazy. Doesn't look like there's any pre-commit hooks in this project to ensure go fmt is run. Probably should have that.

**Pull request merged**